src/main/java/de/rabea/ui/UserInterface.java

```diff
@@ -20,14 +20,21 @@ public UserInterface(Console console) {
     private String unavailablePosition = "Sorry, this position is not available!";
     private String enterANumber = "Please enter a number between 1 and 9.";
     private InputFormatter inputFormatter = new InputFormatter();
+    private String clearScreen= "\033[H\033[2J";
 
 
     public void displayBoard(Board board) {
+        clearTheScreen();
         BoardPainter boardPainter = new BoardPainter(board);
         console.print(boardPainter.drawBoard());
     }
 
+    private void clearTheScreen() {
```

Comment:
asymmetry between this method and the previous. Either it's <verb>Board or <verb>TheBoard.